wgsl/index.bs

```diff
@@ -11734,18 +11735,31 @@ See [[#builtin-inputs-outputs]] for how to specify that a pipeline input or outp
       <td>output
       <td>vec4&lt;f32&gt;
       <td style="width:50%">Output position of the current vertex, using homogeneous coordinates.
-      After homogeneous normalization (where each of the *x*, *y*, and *z* components
-      are divided by the *w* component), the position is in the WebGPU normalized device
-      coordinate space.
+
+      Let the result be the vector (*x*,*y*,*z*,*w*).
+      Then *w* is known as the <dfn noexport>homogeneous divisor</dfn>, and
```

Comment:
nit: `dfn` type dfns are not exported by default https://speced.github.io/bikeshed/#dfn-export

Reply:
Heh.  I put those in by habit when drafting because I think bikeshed errors if it's not referenced.

It would be a healthy exercise to explicitly remove the noexport on everything and see what is unreferenced.

Reply:
I didn't realize noexport suppressed that warning! I should go through and move the API spec's "temporary usages of non-exported dfns" over to noexport.